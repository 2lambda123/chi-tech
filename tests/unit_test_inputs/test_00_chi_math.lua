chiUnitTests_Test_chi_math()