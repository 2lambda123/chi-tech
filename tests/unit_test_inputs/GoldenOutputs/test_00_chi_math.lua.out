[0]  Parsing argument 1 tests/unit_test_inputs/test_00_chi_math.lua
[0m[0]  Parsing argument 2 --supress_beg_end_timelog
[0m[0]  ChiTech number of arguments supplied: 2
[0m[0]  Testing chi_math::DynamicVector
[0m[0]  [1 1 1 1 1]
[0m[0]  Testing chi_math::DynamicMatrix
[0m[0]  1 1 1 1 1 1 1
[0m[0]  1 1 1 1 1 1 1
[0m[0]  1 1 1 1 1 1 1
[0m[0]  1 1 1 1 1 1 1
[0m[0]  1 1 1 1 1 1 1
[0m[0]  ----- SparseMatrix::PrintS() -----
[0m[0]   1.00e+00  1.10e+00  1.20e+00  1.30e+00 
[0m[0]   1.90e+00  2.00e+00  2.10e+00         0 
[0m[0]          0  2.90e+00  3.00e+00  3.10e+00 
[0m[0]          0         0  3.90e+00  4.00e+00 
[0m[0]  
[0m[0]  ----- for (const auto& entry : m.Row(2)) -----
[0m[0]  2 1 2.9
[0m[0]  2 2 3
[0m[0]  2 3 3.1
[0m[0]  ----- after value*2 -----
[0m[0]  2 1 5.8
[0m[0]  2 2 6
[0m[0]  2 3 6.2
[0m[0]  ----- for (auto entry : matrix) -----
[0m[0]  0 0 1
[0m[0]  0 1 1.1
[0m[0]  0 2 1.2
[0m[0]  0 3 1.3
[0m[0]  1 0 1.9
[0m[0]  1 1 2
[0m[0]  1 2 2.1
[0m[0]  2 1 5.8
[0m[0]  2 2 6
[0m[0]  2 3 6.2
[0m[0]  3 2 3.9
[0m[0]  3 3 4
[0m[0]  ----- after compress -----
[0m[0]  0 0 1
[0m[0]  0 1 1.1
[0m[0]  0 2 1.2
[0m[0]  0 3 1.3
[0m[0]  1 0 1.9
[0m[0]  1 1 2
[0m[0]  1 2 2.1
[0m[0]  2 1 5.8
[0m[0]  2 2 6
[0m[0]  2 3 6.2
[0m[0]  3 2 3.9
[0m[0]  3 3 4
[0m